cmd/kubelet/app/options/options.go

```diff
@@ -29,6 +30,26 @@ import (
 	"github.com/spf13/pflag"
 )
 
+const (
+	defaultRootDir             = "/var/lib/kubelet"
+	experimentalFlannelOverlay = false
+
+	// When these values are updated, also update test/e2e/framework/util.go
+	defaultPodInfraContainerImageName    = "gcr.io/google_containers/pause"
+	defaultPodInfraContainerImageVersion = "3.0"
+	// Auto detect cloud provider.
+	AutoDetectCloudProvider         = "auto-detect"
+	DefaultKubeletPodsDirName       = "pods"
+	DefaultKubeletVolumesDirName    = "volumes"
+	DefaultKubeletPluginsDirName    = "plugins"
+	DefaultKubeletContainersDirName = "containers"
```

Comment:
The consts that were moved are used for constructing defaults for the v1alpha1 KubeletConfiguration object. Just glancing through the changes, it seems like the hardcoded const "defaults" are always used. Does "default" imply that it could be possible, in the future, for a user to configure these values? If so, maybe it makes sense to expose these dir names via componentconfig?